src/db/params.rs

```diff
@@ -99,6 +99,8 @@ bso_data! {
     GetBsoTimestamp {},
 }
 
+pub struct Check {}
```

Comment:
nit: this could be killed (have `check` take no args). Is there plans for it in the future?

Reply:
Not really. For consistency (and a bit of sloth) I was using the  async to sync macros which require a matching param and result pair.

Reply:
Ah right, I was thinking we have a few trait methods that take no args (e.g. commit). I forgot they can't/don't use the macro

Reply:
No big, I removed the function from the macros. If we ever need to add input params, we can swap it back.